### PR TITLE
Refactor stamp_generation/

### PR DESCRIPTION
**Pull request by KomalBhadani27**
- Removed factory files
- Restructured stamp_generator

### CODE REVIEW

File: stamp_generation/stamp_generator.py [hunk 1]
```diff
@@ -2,25 +2,45 @@
...
     """This helps in creation of stamp_pages"""
 
     def __init__(self, _website, stamp_pages):
         self._website = _website
         self.stamp_pages = stamp_pages
+        self.stamp_file_path = self.generate_stamp_file()
 
-    def generate_stamp(self):
+    def generate_stamp_file(self):
         """This method uses jinja2 template(stamp_template.html.jinja) and renders
```
Comment (detel) [inline stamp_generation/stamp_generator.py:25]: Update this method's comment.

Comment (detel) [reply stamp_generation/stamp_generator.py:25]: Explain what is returns now.

File: stamp_generation/stamp_generator.py [hunk 2]
```diff
@@ -2,25 +2,45 @@
...
-        template = Template(open(template_path).read())
-        return template.render(
-            publisher_domain=self._website.domain,
-            logo_url=self._website.logo_url,
-            canonical_url=self._website.url,
-            contents=self._website.contents.content_list,
-            pages=self.stamp_pages.stamp_pages)
+        template = Template(open(STAMP_TEMPLATE_PATH).read())
+
+        final_stamp_file \
```
Comment (detel) [inline stamp_generation/stamp_generator.py:30]: You can remove `final_` prefix.
`stamp_file_name` or `stamp_file` is fine.

File: stamp_generation/stamp_generator.py (hunk 1 above)
Comment (KomalBhadani27) [reply stamp_generation/stamp_generator.py:25]: My bad! Will update it.

File: generated_stamp/Despite_Guaranteeing_AAP's_Landslide_Victory,_There_Are_Sadly_No_Women_On_Its_Cabinet._Why?_stamp.html [hunk 3]
```diff
@@ -10,12 +10,12 @@
     <script async custom-element="amp-youtube" src="https://cdn.ampproject.org/v0/amp-youtube-0.1.js"></script>
     <script async custom-element="amp-instagram" src="https://cdn.ampproject.org/v0/amp-instagram-0.1.js"></script>
     <script async custom-element="amp-pinterest" src="https://cdn.ampproject.org/v0/amp-pinterest-0.1.js"></script>
-    <title>This is title of website</title>
```
Comment (KomalBhadani27) [inline generated_stamp/Despite_Guaranteeing_AAP's_Landslide_Victory,_There_Are_Sadly_No_Women_On_Its_Cabinet._Why?_stamp.html:13]: This file can be ignored while review. The name of the file follows the format **title_stamp.html** where if title had spaces, it was replaced with _.

Comment (KomalBhadani27) [inline generated_stamp/Despite_Guaranteeing_AAP's_Landslide_Victory,_There_Are_Sadly_No_Women_On_Its_Cabinet._Why?_stamp.html:13]: This file can be ignored while review. The file name is changed to format **title_stamp.html** and moved to `stampify/generated_stamp/`.

Comment (detel) [reply generated_stamp/Despite_Guaranteeing_AAP's_Landslide_Victory,_There_Are_Sadly_No_Women_On_Its_Cabinet._Why?_stamp.html:13]: Why do we need to merge this?

Comment (KomalBhadani27) [reply generated_stamp/Despite_Guaranteeing_AAP's_Landslide_Victory,_There_Are_Sadly_No_Women_On_Its_Cabinet._Why?_stamp.html:13]: Ahh, good question. We don't need to. I will remove it. Thanks!

**Pull request merged**